tsl/src/bgw_policy/compression_api.c

```diff
@@ -65,15 +65,6 @@ policy_compression_get_maxchunks_per_job(const Jsonb *config)
 	return (found && maxchunks > 0) ? maxchunks : 0;
 }
 
-bool
-policy_compression_get_verbose_log(const Jsonb *config)
-{
```

Comment:
Since you are adding this change (i.e. verbose_log in config ) for retention policy etc, it makes more sense to move this function to utils.c, instead of removing this code.
Rename the function to something more generic: policy_get_verbose_log when you move it to utils.c